x-pack/packages/kbn-elastic-assistant/impl/assistant/quick_prompts/quick_prompt_settings/quick_prompt_editor.tsx

```diff
@@ -177,6 +175,17 @@ const QuickPromptSettingsEditorComponent = ({
     ]
   );
 
+  const setDefaultPromptColor = useCallback((): string => {
+    const randomColor = getRandomEuiColor();
+    handleColorChange(randomColor, { hex: randomColor, isValid: true });
+    return randomColor;
+  }, [handleColorChange]);
```

Comment:
I've been wanting to do this for a while now, random EuiColors for quick prompt colors:

https://github.com/user-attachments/assets/8e91f7e6-58a7-4381-ac10-16016890dce2